src/components/Attachments/AttachmentView/AttachmentViewPdf/BaseAttachmentViewPdf.js

```diff
@@ -1,9 +1,34 @@
 import React, {memo, useCallback, useContext, useEffect} from 'react';
+import PropTypes from 'prop-types';
 import {attachmentViewPdfPropTypes, attachmentViewPdfDefaultProps} from './propTypes';
 import PDFView from '../../../PDFView';
 import AttachmentCarouselPagerContext from '../../AttachmentCarousel/Pager/AttachmentCarouselPagerContext';
 
-function AttachmentViewPdf({file, encryptedSourceUrl, isFocused, isUsedInCarousel, onPress, onScaleChanged: onScaleChangedProp, onToggleKeyboard, onLoadComplete, errorLabelStyles, style}) {
+const propTypes = {
+    /** Updates the scale value of the pdf */
+    updateScaleRef: PropTypes.func,
```

Comment:
I don't feel this is the correct name. We usually use `ref` prefix for passing the ref.

Reply:
Yeah, that sounds better.